ReactNativeClient/lib/models/Setting.js

```diff
@@ -661,7 +666,24 @@ class Setting extends BaseModel {
 
 				this.cache_.push(c);
 			}
+			if (this.constants_.appType !== 'mobile') this.loadSecureKeys();
+			this.dispatchUpdateAll();
```

Comment:
Please don't check for app type here. Instead use lib/shim.js to abstract away differences between platforms. For example create a function in the shim that returns all the secure credentials. On Node/Electron, make it use keytar. On React Native, since it's not implemented, make it return an empty array.

Make sure that the logic means that it works both on RN and Node.

Reply:
I wasn't aware of the shim.js file. I'll make use of it now!